ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/TransactionValidatorFactory.java

```diff
@@ -30,6 +31,7 @@
 public class TransactionValidatorFactory {
 
   private volatile Supplier<TransactionValidator> transactionValidatorSupplier;
+  private static final Set<BlobType> NO_BLOB = Set.of();
```

Comment:
maybe rename to this
```suggestion
  private static final Set<BlobType> NO_BLOB_ACCEPTED = Set.of();
```

Reply:
BLOBS_PROHIBITIED